themes/classic/templates/_partials/form-errors.tpl

```diff
@@ -27,7 +27,7 @@
     {block name='form_errors'}
       <ul>
         {foreach $errors as $error}
-          <li class="alert alert-danger">{$error}</li>
+          <li class="alert alert-danger">{$error|strip_tags|nl2br nofilter}</li>
```

Comment:
why strip_tags?

Reply:
to be sure no html tag is inserted (since I use `nofilter`) only the converted `\n` remain

Reply:
I can remove it, it will still work but I was worried of XSS injection 😛

Reply:
only if someone include xss in translations, but we should be worried if it happens ^^

Reply:
alright I'll remove it then

Reply:
Done